plugin.yml

```diff
@@ -1,10 +1,10 @@
 name: Slapper
 author: jojoe77777
-version: 1.5.3
+version: 1.5.4
```

Comment:
might as well bump to 1.6.0 since the patch numbers for 1.5 are broken on poggit